src/EnergyPlus/UnitarySystem.cc

```diff
@@ -125,49 +125,17 @@ namespace UnitarySystems {
                               bool const ZoneEquipment,
                               Real64 &sysOutputProvided,
                               Real64 &latOutputProvided)
-    {
-        simulateSys(state,
-                    Name,
-                    FirstHVACIteration,
-                    AirLoopNum,
-                    CompIndex,
-                    HeatActive,
-                    CoolActive,
-                    ZoneOAUnitNum,
-                    OAUCoilOutTemp,
-                    ZoneEquipment,
-                    sysOutputProvided,
-                    latOutputProvided);
-    }
-
-    void UnitarySys::simulateSys(EnergyPlusData &state,
-                                 std::string_view Name,
-                                 bool const FirstHVACIteration,
-                                 int const AirLoopNum,
-                                 int &CompIndex,
-                                 bool &HeatActive,
-                                 bool &CoolActive,
-                                 int const ZoneOAUnitNum,
-                                 Real64 const OAUCoilOutTemp,
-                                 bool const ZoneEquipment,
-                                 Real64 &sysOutputProvided,
-                                 Real64 &latOutputProvided)
     {
         DataHVACGlobals::CompressorOperation CompressorOn = DataHVACGlobals::CompressorOperation::Off;
 
         // Obtains and Allocates unitary system related parameters from input file
         if (this->m_ThisSysInputShouldBeGotten) {
```

Comment:
I'm not going to harp on this too much because it's not super important, but ... `this->` and `m_` are redundant. 

`m_` is "scoping Hungarian notation" to indicate that a variable has object member scope, i.e., it's a member of the object on which the current method is being called, i.e., a member of `this`, i.e., equivalent `this->m_`. The two other scopes are local (declared and live within the function, this includes parameters) and global. In "scoping Hungarian", global variables begin with `g_` and local variables don't have a prefix at all.

The C/C++ scoping precedence rule is local > member > global, i.e., the compiler will first look for a variable among the list of local variables, if there is no match there it will look for it in the list of member variables (and if it finds it there it will implicitly add the `this->`), finally it will look in the list of global variables. The only time `this->` is really necessary is if there is a local variable and a member variable of the same name and you want to refer to the member variable. However, if all member variables start with `m_` and all local variables do not then this situation never comes up. 

TL;DR If you are already using `m_` for member variables (which I think is good practice), you don't need `this->`. Using `this->` doesn't make the code slower, just more verbose and IMO difficult to read.